src/users/fixtures/list-users.json

```diff
@@ -0,0 +1,25 @@
+{
+  "object": "list",
+  "data": [
+    {
+      "object": "user",
+      "id": "user_01H5JQDV7R7ATEYZDEG0W5PRYS",
+      "email": "test01@example.com",
+      "first_name": "Test 01",
+      "last_name": "User",
+      "identities": [],
+      "created_at": "2023-07-18T02:07:19.911Z",
+      "updated_at": "2023-07-18T02:07:19.911Z",
+      "user_type": "unmanaged",
+      "email_verified_at": "2023-07-17T20:07:20.055Z"
+    }
+  ],
+  "list_metadata": {
+    "before": null,
+    "after": null
+  },
+  "listMetadata": {
+    "before": null,
+    "after": null
+  }
```

Comment:
I believe this is no longer a thing after workos/workos#21671.

Reply:
Yep, we'll need to make sure the SDK doesn't expect the legacy pagination metadata field, since it won't be sent for these new APIs.

Reply:
thanks, my branch was out of date. fixed.